docs/reference/repositories-stats/apis/get-repositories-stats.asciidoc

```diff
@@ -0,0 +1,142 @@
+[[get-repositories-stats-api]]
+=== Repositories statistics API
+++++
+<titleabbrev>Repositories statistics</titleabbrev>
+++++
+
+Returns cluster repositories utilization statistics.
+
+[[get-repositories-stats-api-request]]
+==== {api-request-title}
+
+`GET /_nodes/<node_id>/_repositories_stats`
+
+[[get-repositories-stats-api-desc]]
+==== {api-description-title}
+
+You can use the cluster repositories statistics API to retrieve repository utilization statistics in a cluster.
+
+
+All the nodes selective options are explained <<cluster-nodes,here>>.
+
+[[get-repositories-stats-api-path-params]]
+==== {api-path-parms-title}
+
+include::{es-repo-dir}/rest-api/common-parms.asciidoc[tag=node-id]
+
+[[get-repositories-stats-api-query-params]]
+==== {api-query-parms-title}
+
+[role="child_attributes"]
+[[get-repositories-stats-api-response-body]]
+==== {api-response-body-title}
+
+`_nodes`::
+(object)
+Contains statistics about the number of nodes selected by the request.
++
+.Properties of `_nodes`
+[%collapsible%open]
+====
+`total`::
+(integer)
+Total number of nodes selected by the request.
+
+`successful`::
+(integer)
+Number of nodes that responded successfully to the request.
+
+`failed`::
+(integer)
+Number of nodes that rejected the request or failed to respond. If this value
+is not `0`, a reason for the rejection or failure is included in the response.
+====
+
+`cluster_name`::
+(string)
+Name of the cluster. Based on the <<cluster.name>> setting.
+
+`nodes`::
```

Comment:
We still don't have a concept of sending this request to a specific node and supplying a `?local=true` flag do we? I think this was requested and makes a lot of sense to have since it would allow for just sending these requests to all the nodes by transport address instead of having to deal with timeouts and whatnot from the combined request?

Reply:
`TransportGetRepositoriesAction` is a subclass of `TransportNodesAction`, so it can resolve `_local` and `_all` `nodeId` references to query the local node or all the nodes transparently.

Reply:
:facepalm: right my bad :)